inc/utilities.h

```diff
@@ -0,0 +1,64 @@
+/* ************************************************************************** */
+/*                                                                            */
+/*                                                        :::      ::::::::   */
+/*   utilities.h                                        :+:      :+:    :+:   */
+/*                                                    +:+ +:+         +:+     */
+/*   By: u413q <u413q@student.42.fr>                +#+  +:+       +#+        */
+/*                                                +#+#+#+#+#+   +#+           */
+/*   Created: 2023/10/29 11:46:04 by u413q             #+#    #+#             */
+/*   Updated: 2023/10/29 17:56:01 by u413q            ###   ########.fr       */
+/*                                                                            */
+/* ************************************************************************** */
+
+#ifndef UTILITIES_H
```

Comment:
Nit: i think "utils" is more common

Reply:
adapted in each occurence and file names .c/.h